docs/geedocs/5.2.5/answer/7160006.html

```diff
@@ -0,0 +1,389 @@
+  <p class="sidebar-toc"><a href="../answer/7160000.html">Open GEE 5.2.0</a></p>
+  <p class="sidebar-toc"><a href="../answer/7159245.html">GEE 5.1.3</a></p>
+  <p class="sidebar-toc"><a href="../answer/6259403.html">GEE 5.1.2</a></p>
+  <p class="sidebar-toc"><a href="../answer/6159443.html">GEE 5.1.1</a></p>
+  <p class="sidebar-toc"><a href="../answer/6078762.html">GEE 5.1.0</a></p>
+  <p class="sidebar-toc"><a href="../answer/6053170.html">GEE 5.0.2</a></p>
+  <p class="sidebar-toc"><a href="../answer/4556704.html">GEE 5.0.1</a></p>
+  <p class="sidebar-toc"><a href="../answer/3424202.html">GEE 5.0</a></p>
+</div>
+<div class="content">
+<a name="top_of_file"></a>
+<p><img src="../art/common/googlelogo_color_260x88dp.png" width="130" height="44" alt="Google logo" /></p>
+<h1><a href="../index.html">Google Earth Enterprise Documentation Home</a> | Release notes</h1>
+<h2>Release notes: Open GEE 5.2.5</h2>
+
+<p>Open GEE 5.2.5 is currently in development</p>
+
+  <h5>
+    <p id="overview_5.2.5">New Features</p>
+  </h5>
+    <p><strong>C++98 no longer supported</strong>. As of 5.2.4 C++11 syntax can be used and building open GEE using C++98 standard will no longer be supported</p>
+
+  <h5>
+    <p id="supported_5.2.5">Supported Platforms</p>
+  </h5>
+  <p>The Open GEE 5.2.5 release is supported on 64-bit versions of the following operating systems:</p>
+    <ul>
+      <li>Red Hat Enterprise Linux version 6.x and 7.x, including the most recent security patches</li>
+      <li>CentOS 6.x and 7.x</li>
+      <li>Ubuntu 14.04 LTS and 16.04 LTS</li>
+    </ul>
+    <p>Google Earth Enterprise 5.2.5 is compatible with Google Earth Enterprise Client (EC) version 7.1.5 and above.</p>
+    <p><p>To upgrade from Open GEE 5.2.0, do NOT uninstall it. We recommend that you upgrade Open GEE 5.2.0 by simply installing Open GEE 5.2.5. Installing Open GEE 5.2.5 on top of Open GEE 5.2.0 will ensure that your PostgreSQL databases are backed up and upgraded correctly to the new PostgreSQL version used by Open GEE 5.2.5.</p></p>
+  <h5>
+    <h5>
+      <p id="resolved_issues_5.2.5">Resolved Issues</p>
+    </h5>
+    <div>
+      <table class="nice-table">
+        <tbody>
+          <tr>
+            <th>Number</th>
+            <th class="narrow">Description</th>
+            <th>Resolution</th>
+          </tr>
+          <tr>
+            <td>934</td>
+            <td>Polygon displayed at the wrong location when viewing GLC assemly</td>
+            <td>
+              The Cutter WSGI script now looks at the projection of the GLM files going into the assembly,
+              rather than using a hardcoded Mercator projection for the assembly.  Also, if a user tries to
+              assemble GLMs with differing projections, a warning will be issued.
+            </td>
+          </tr>
+        </tbody>
+      </table>
+  <h5>
+  <h5>
+    <p id="known_issues_5.2.5">Known Issues</p>
+  </h5>
```

Comment:
Should the items in the Known/Resolved Issues lists be removed from this page since this is where future 5.2.5 work will be documented?

Reply:
Known issues carry over from version to version unless they are fixed